packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -299,6 +303,66 @@ const assertNoImportedCssVariables = (
   }
 };
 
+/**
+ * Manipulates template literal so that it resembles
+ * CSS `property: value` declaration where
+ * - before: all the CSS _before_ the interpolation
+ * - after: all the CSS _after_ the interpolation
+ * @param expression
+ * @param quasi
+ */
+const moveCssPropertyInExpression = (
+  expression: t.ConditionalExpression,
+  quasi: t.TemplateElement,
+  nextQuasis: t.TemplateElement,
+  deleteValue = true
+): void => {
+  const conditionalPaths: ['consequent', 'alternate'] = ['consequent', 'alternate'];
+  const value = quasi.value.raw;
+  const re = /([^;/]+$)/; // everything after the last ';'
```

Comment:
Can you use `split` for this too? It should be more performant and readable than a regex

Reply:
I forgot to mention that this particular regex was a quick way to make things work. I am not 100% sure whether we should rely on the semicolon; what if we have a semicolon within the CSS value ( eg background: url ("this contains a semicolon")

- [ ]  try not to use regex